tests/conftest.py

```diff
@@ -72,6 +73,14 @@ def db(astra_db_credentials_kwargs: Dict[str, Optional[str]]) -> AstraDB:
     return AstraDB(**astra_db_credentials_kwargs)
 
 
+@pytest.fixture
```

Comment:
We can't make it session-scoped because of https://github.com/pytest-dev/pytest-asyncio/issues/706#issuecomment-1838954312